add poi-label gazetteer

aparlato opened the pull request:
Adds poi-label gazetteer.

cc @nickidlugash

nickidlugash commented on mapbox-streets/poi-label.json:
```diff
@@ -0,0 +1,462 @@
+ {
+    "type": "FeatureCollection",
+    "name": "poi-label",
```
Perhaps make the name more specific?

nickidlugash commented on mapbox-streets/poi-label.json:
```diff
@@ -0,0 +1,462 @@
+            "properties": {
+                "place_name": "Presidio and San Francisco National Cemetery, USA",
+                "zoom": 16,
+                "description": "Two low scalerank POIs next to one another surrounded by higher scalerank POIs.",
```
scalerank --> sizerank?

nickidlugash commented on mapbox-streets/poi-label.json:
```diff
@@ -0,0 +1,462 @@
+            "properties": {
+                "place_name": "San Antonio State Hospital, San Antonio TX, USA",
+                "zoom": 15,
+                "description": "San Antonio State Hospital.",
```
The description field is optional – maybe don't include one if it's not going to have any more information that isn't already in the name? Ideally though I think we use this field: https://github.com/mapbox/gazetteer/blob/gazetteer-3/CONTRIBUTING.md#description

aparlato commented:
@nickidlugash I think I've addressed all of your comments now!

> Many (but not all) of the highlights in this gazetteer include the id data field – is there a particular reason you wanted to include this for some of the features?

This was a leftover since I started this gazetteer before writing in the code that deleted the `id` field. We need the `id` field while creating the entry, but once the entry is created we delete this now in carto-cam since it's no longer necessary.

nickidlugash commented:
@aparlato LGTM!
